src/frequency_compensation_filter.h

```diff
@@ -0,0 +1,55 @@
+/* -*- c++ -*- */
+/*
+ * Copyright 2019 Analog Devices Inc.
+ *
+ * This is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3, or (at your option)
+ * any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software; see the file COPYING.  If not, write to
+ * the Free Software Foundation, Inc., 51 Franklin Street,
+ * Boston, MA 02110-1301, USA.
+ */
+
+#ifndef INCLUDED_BLOCKS_OVERSHOOT_FILTER_H
+#define INCLUDED_BLOCKS_OVERSHOOT_FILTER_H
+
+#include <gnuradio/blocks/api.h>
+#include <gnuradio/sync_block.h>
+
+namespace adiscope {
+
+    class frequency_compensation_filter : virtual public gr::sync_block
+    {
+    public:
+      // gr::blocks::streams_to_vector::sptr
+      typedef boost::shared_ptr<frequency_compensation_filter> sptr;
+
+      /*!
+       * Make a stream-to-vector block.
```

Comment:
Is this comment related to the GR block?

Reply:
fixed